public/index.html

```diff
@@ -14,5 +14,8 @@
     </noscript>
     <div id="app"></div>
     <!-- built files will be auto injected -->
+    <!-- Fathom - beautiful, simple website analytics -->
+    <script src="https://cdn.usefathom.com/3.js" site="RZIPTLKO"></script>
```

Comment:
Eventually the site parameter needs to be dynamic, otherwise we would not be able to run analytics on beta / stage / prod at the same time.